python/src/Student_doc.i.in

```diff
@@ -109,48 +109,45 @@ distribution= Student(nu = 3, mu = [0,0,0,0,0], sigma = [1.5,0.4,1,2.5,1], R = 5
  [ -0.5  0    1    0    0   ]
  [  0    0    0    1   -0.2 ]
  [  0    0    0   -0.2  1   ]])"
-
-// ---------------------------------------------------------------------
-
-%feature("docstring") OT::Student::getMu
-"Accessor to the distribution's mean.
-
-Returns
--------
-mu : float
-    Mean.
-    Only defined when the dimension is 1
-    (else, use the :func:`getMean()` method inherited from the
-    EllipticalDistribution class)."
-
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::Student::getNu
-"Accessor to the distribution's generalised number degrees of freedom.
+"Accessor to the generalized number degrees of freedom.
 
 Returns
 -------
 nu : float
-    Generalised number degrees of freedom."
+    The :math:`\nu` parameter."
```

Comment:
This is strange : in setNu you make the opposite change.

Reply:
ok. I make a choice.